### ISSUE TITLE
support pkgs: check for 'solid' in binary STL headers

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/abb#104]
When loading the meshes provided as part of the `abb_irb2400_support` package, I get multiple warnings on the console:

```
[1463395556.713573853] [ WARN] [ros.rviz]: The STL file 'package://abb_irb2400_support/meshes/irb2400/visual/link_1.stl'
is malformed. It starts with the word 'solid', indicating that it's an ASCII STL file, but
it does not contain the word 'endsolid' so it is either a malformed ASCII STL file or it
is actually a binary STL file. Trying to interpret it as a binary STL file instead.
```

Probably exported using SolidWorks, which tends to do this.

As the warning says, having `solid` in the header confuses loaders, so it should be removed.

**Comment by gavanderhoorn** [comment]
Appears to be a problem with `5400_support` as well.

**Comment by gavanderhoorn** [comment]
Adressed in #111 and #112.

**Comment by gavanderhoorn** [comment]
Resolved in 9fb3c0c7a0279901bfab4031923afbf234bec27f and e669435ec3aa47d271b11e1f9e1e18c91eb6d674.
